addons/event/models/event_event.py

```diff
@@ -176,6 +176,20 @@ def _default_event_mail_ids(self):
         'Start sale date', compute='_compute_start_sale_date',
         help='If ticketing is used, contains the earliest starting sale date of tickets.')
 
+    def name_get(self):
+        if not self.env.context.get('registration'):
```

Comment:
Would use a more precise naming. Grepping "registration" in the PR to try to find where it is used already gives 84 results; post merge this would be very difficult to track. Something like ``registration_name_with_seats" ?